sys/include/net/nanocoap.h

```diff
@@ -263,6 +276,29 @@ typedef struct {
 } coap_resource_t;
 
 /**
+ * @brief   Block1 helper struct
+ */
+typedef struct {
+    size_t offset;                  /**< offset of received data            */
+    uint32_t blknum;                /**< block number                       */
+    unsigned szx;                   /**< szx value                          */
+    int more;                       /**< -1 for no option, 0 for last block,
+                                          1 for more blocks coming          */
+} coap_block1_t;
+
+/**
+ * @brief Blockwise transfer helper struct
+ */
+typedef struct {
+    size_t start;                   /**< Start offset of the current block  */
+    size_t end;                     /**< End offset of the current block    */
+    size_t cur;                     /**< Offset of the  generated content   */
+    uint8_t *opt;                   /**< Pointer to the placed option       */
+} coap_blockhelper_t;
+
+/**
+>>>>>>> 01c4c5265... nanocoap: Add server side block2 support
+>>>>>>> nanocoap: Add server side block2 support
```

Comment:
Another one ;-).

Reply:
First :coffee:, otherwise half this PR is going to consist of me removing merge conflict indicators :disappointed:

Reply:
I hope they're all gone now